include/secp256k1.h

```diff
@@ -394,10 +396,10 @@ SECP256K1_API int secp256k1_ecdsa_signature_parse_compact(
 /** Parse a DER ECDSA signature.
  *
  *  Returns: 1 when the signature could be parsed, 0 otherwise.
- *  Args: ctx:      a secp256k1 context object
- *  Out:  sig:      a pointer to a signature object
- *  In:   input:    a pointer to the signature to be parsed
- *        inputlen: the length of the array pointed to be input
+ *  Args: ctx:      a secp256k1 context object.
+ *  Out:  sig:      a pointer to a signature object.
+ *  In:   input:    a pointer to the signature to be parsed.
+ *        inputlen: the length of the array pointed to be input.
```

Comment:
Adding periods for every arguments seems a bit unnecessary. So far I've tried to follow the rule to add a period only if it follows an actual sentence (obj + verb).

Reply:
I mostly wanted consistency and thought that will be the less controversial way, but I'll remove that change

Reply:
I removed all the changes that simply added periods